test/fuzz/test_runner.py

```diff
@@ -100,10 +100,12 @@ def main():
 
 def run_once(*, corpus, test_list, build_dir, export_coverage):
     for t in test_list:
+        corpus_path = os.path.join(corpus, t)
+        os.makedirs(corpus_path, exist_ok=True)
```

Comment:
See the documentation: "The seed corpus to run on (must contain subfolders for each fuzz target)"

the idea was to give the user feedback when the seeds were missing by accident

Reply:
The consequence of that is that Travis fails when adding a new fuzzer (like in this case prior to adding that change). That's not how we want things to work, right? :)

Reply:
I think we do want this (as a reminder to generate seeds and upload them for travis)

Reply:
I think that creates a inter-repo dependency that is not warranted: the build for https://github.com/bitcoin/bitcoin should not break due to https://github.com/bitcoin-core/qa-assets not being updated. I suggest moving this checking to a cron job or to https://github.com/bitcoin-core/qa-assets.

Our fuzzing framework is currently in a really really poor state: to fix that we want to make it super easy to add new fuzzers. Removing this inter-repo dependency is a quick way to make it easier to add new fuzzers without any measurable cost: modern coverage-guided fuzzers such as `libFuzzer` usually provide good value even in the absence of a starting corpus.

Reply:
I'm now using suppressions in `fuzz/test_runner.py` to resolve this.